requirements.txt

```diff
@@ -1,2 +1,2 @@
 ovos-utils~=0.0, >=0.0.28a4
-ovos_workshop~=0.0, >=0.0.11a4
\ No newline at end of file
+ovos_workshop~=0.0, >=0.0.12a8
```

Comment:
Does this not work with the older version of workshop? I feel we should keep the oldest working version as the requirement